.github/workflows/docker.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       CONTAINER_REGISTRY: ghcr.io
       IMAGE_TAG: ${{ github.ref_name }}
     runs-on: ubuntu-22.04
-    container: rust:1.72
+    container: rust:1.77.0
```

Comment:
why not `1.77.2` which is used in libtelio (this applies to all other similar places)?

Reply:
`1.77.0` on `nightly-2024-01-25` wre the latest versions I could confirm compatibility with the `dylint` crate